src/python/nimbusml/internal/core/linear_model/linearsvmbinaryclassifier.py

```diff
@@ -69,7 +69,7 @@ class LinearSvmBinaryClassifier(
 
     :param caching: Whether trainer should cache input training data.
 
-    :param lambda_: Regularizer constant.
+    :param regularization: Regularizer constant.
```

Comment:
These 3 files are auto generated from manifest.json. You could use manifest_diff.json to specify renaming of the parameter then re-run entrypoint_compiler tools with generate options set to True. There are many examples in manifest_diff.json how to do renaming.